### PR TITLE
Ui Tweaks

### PR DESCRIPTION
**Pull request by taylorpoe**
**Has been reviewed/approved:**
* Updates various small icon, coloring, & layout details
* Updates css transition group package from the old deprecated package to the suggested one [instructions on the package we were using](https://www.npmjs.com/package/react-addons-css-transition-group)
* Disallow project creation if name is blank

**Still needs review:**
* Only show the actions bolt icon next to collapsed element rows names if it _has_ actions attached. 
* Fixes actions icons to change to blue immedately when elements are given actions
* put ticker line above collapsed rows
* Fixes click area of frame events to not respond to gauge numbers being clicked
* Brings ticker bulb up on top of numbers to reduce overall gauge height — had to put the numbers back in the bulb because the bulb now covers the numbers.

### CODE REVIEW

**Comment by stristr** [comment]
Deps management LGTM, have not reviewed code.

File: packages/haiku-creator/src/react/components/ProjectBrowser.js [hunk 1]
```diff
@@ -240,6 +245,7 @@ class ProjectBrowser extends React.Component {
 
   handleNewProjectGo () {
     const raw = this.newProjectInput.value
+    if (!raw) return false
```
Comment (tsortwehttam) [inline packages/haiku-creator/src/react/components/ProjectBrowser.js]: What if the string is `"    "` or `"\n   "`?

File: packages/haiku-creator/public/stylesheets/global.css [hunk 2]
```diff
@@ -264,8 +264,8 @@ input::-webkit-input-placeholder {
 }
 
 .toast-enter {
-  opacity: .1;
-  transform: translateY(-400px);
+  opacity: .01;
```
Comment (roperzh) [inline packages/haiku-creator/public/stylesheets/global.css]: _extreme nit_: `0.01` reads much more clearly for me, but whatever we choose should be consistent across the file.

File: packages/haiku-creator/src/react/components/ProjectBrowser.js (hunk 1 above)
Comment (roperzh) [inline packages/haiku-creator/src/react/components/ProjectBrowser.js]: _nit_: would prefer a more explicit name like `rawProjectName` or similar

**Review by roperzh — CHANGES_REQUESTED** [review]
LGTM, the only necessary change to make is a lint pass (`yarn lint-all`), as there are unused variables and other lint issues

File: packages/haiku-creator/src/react/components/ProjectBrowser.js (hunk 1 above)
Comment (tsortwehttam) [reply packages/haiku-creator/src/react/components/ProjectBrowser.js]: That bad variable name is my doing and I agree it should be changed.

Comment (taylorpoe) [reply packages/haiku-creator/src/react/components/ProjectBrowser.js]: Spaces and slashes are both ignored as the user is typing; so they won't ever end up here.

Comment (taylorpoe) [reply packages/haiku-creator/src/react/components/ProjectBrowser.js]: updated

File: packages/haiku-creator/public/stylesheets/global.css (hunk 2 above)
Comment (taylorpoe) [reply packages/haiku-creator/public/stylesheets/global.css]: updated

**Review by roperzh — APPROVED** [review]

**Comment by roperzh** [comment]
:shipit:

**Comment by taylorpoe** [comment]
Oops, I incorrectly thought this was already merged when I saw that the PR was approved. I have a few new commits here that will need to be reviewed/approved. I will outline the full scope in the PR message.

**Comment by tsortwehttam** [comment]
LGTM.

**Comment by roperzh** [comment]
LGTM.

**Pull request merged**